modules/EvseV2G/iso_server.cpp

```diff
@@ -197,24 +198,22 @@ static bool load_contract_root_cert(mbedtls_x509_crt *contract_root_crt, const c
  * \param flags
  * \return
  */
-static int debug_verify_cert( void *data, mbedtls_x509_crt *crt, int depth, uint32_t *flags )
-{
+static int debug_verify_cert(void* data, mbedtls_x509_crt* crt, int depth, uint32_t* flags) {
     char buf[1024];
-    ((void) data);
+    ((void)data);
```

Comment:
I guess this is because of compiler warnings, should add a comment to be more verbose, like `// unused variables`